src/Savebar/Savebar.tsx

```diff
@@ -1,5 +1,5 @@
-import Button from "@material-ui/core/Button";
 import React from "react";
+import { Button } from "..";
```

Comment:
We should use some absolute path for main catalog, will make it easier if we ever change the component structure or nest it level deeper

Reply:
There's a ticket for this in "technical debt" list